components/admin/BearerRefresh/BearerRefresh.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button } from "@components/forms";
+import { logMessage } from "@lib/logger";
+import { BearerResponse, FormDBConfigProperties } from "@lib/types";
+import axios from "axios";
+import server from "i18next-hmr/server";
+import { useTranslation } from "next-i18next";
+import React, { useEffect, useState } from "react";
+
+export interface BearerRefreshProps {
+  form?: FormDBConfigProperties;
+}
+
+const BearerRefresh = (props: BearerRefreshProps): React.ReactElement => {
+  const { form } = props;
+  const [bearerTokenState, setBearerTokenState] = useState("");
+  const { t } = useTranslation("admin-templates");
+  const [errorState, setErrorState] = useState({ message: "" });
+
+  useEffect(() => {
+    setBearerTokenState(form?.bearerToken ? form.bearerToken : "");
+  }, []);
+
+  /**
+   * Connects to the database and refreshes the bearer token for the specified form ID.
+   *
+   * @param formID
+   */
+  const handleRefreshBearerToken = async (formID: number | undefined) => {
+    try {
+      setErrorState({ message: "" });
+      const serverResponse = await axios({
+        url: `/api/id/${formID}/bearer`,
+        method: "POST",
+        timeout: 0,
+      });
+      const { bearerToken } = serverResponse.data as unknown as BearerResponse;
+      setBearerTokenState(bearerToken);
```

Comment:
This can also go back to our conversation the other day where the `form.bearerToken` prop has essentially been mutated but the parent components are not aware of the change.  Will work for now but ideally the `useRefesh` hook could be used here to ensure state remains constant across all components that use the 'form' prop.

Reply:
Changed it so that the props contain just the "formID", and is never updated.